server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -260,25 +329,20 @@ public void getUsersByCompanyAndAge() throws IOException {
 
   @Test
   public void getUserWithExistentId() throws IOException {
-
     String testID = samsId.toHexString();
+    Context ctx = mockContext("api/users/:id", Map.of("id", testID));
```

Comment:
Just to check, should we also change the paths `"api/users/{id}"` in the test file?

Reply:
Oh, probably. As far as I can tell what you put here in the path is actually ignored unless your code uses it. So I'm not sure if we want `/{id}` or the actual ID or…

We definitely don't want `:id`, though.